normandy/settings.py

```diff
@@ -80,8 +78,12 @@ class Core(Configuration):
         'django.contrib.staticfiles.finders.FileSystemFinder',
         'django.contrib.staticfiles.finders.AppDirectoriesFinder',
         'npm.finders.NpmFinder',
-        'pipeline.finders.PipelineFinder',
     ]
+
+    STATICFILES_DIRS = (
```

Comment:
this should probably move into the `Base` class with `STATIC_ROOT` and `STATIC_URL`